src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/MultiSelectList.cs

```diff
@@ -10,6 +10,11 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Represents a list that lets users select one item.
+    /// This class is typically rendered as an HTML <code>&lt;select&gt;</code> element with the specified collection
+    /// of <see cref="SelectListItem"/> objects.
```

Comment:
Perhaps worth mentioning the `<select>` element will usually have a `multiple="multiple"` attribute.